examples/streaming_transcribe_from_local_file.js

```diff
@@ -27,8 +27,15 @@ client.on('connect', connectionMessage => {
     console.log(`Connected with job id: ${connectionMessage.id}`);
 })
 
+// Optional config to be provided.
+const sessionConfig = new revai.SessionConfig(
+    /* (optional) metadata */ "this is an example", 
```

Comment:
I have not seen the prefixed comment documentation before. is this standard? I would default to putting the comments on the end of the lines.

Reply:
```suggestion
    /* (optional) metadata */ "my example metadata string", 
```

Reply:
Given that both you and @k-weng expressed this sentiment, I'll change this.